src/main/java/com/artipie/rpm/Repomd.java

```diff
@@ -260,6 +198,31 @@ private static Completable gzip(final Path input, final Path output) {
             });
     }
 
+    /**
+     * Result from storage.
+     *
+     * @param nodes XML node
+     * @param file Path of file
+     * @param rxsto RXTO wrapper
+     * @return Result from storage
+     */
+    private static Completable resultFromStorage(
```

Comment:
@Guseyn the name is too vague here, this method loads metadata file by location name in `repomd.xml`, maybe we can name it `metaFile()` which will return `Single<Path>` instead of using `Path file` parameter?

Reply:
@g4s8 fixed.